pkg/kubevirt/core/util.go

```diff
@@ -57,10 +60,84 @@ func encodeProviderID(machineID string) string {
 	return fmt.Sprintf("%s/%s", ProviderName, machineID)
 }
 
+func buildNetworks(networkSpecs []api.NetworkSpec) ([]kubevirtv1.Interface, []kubevirtv1.Network, string) {
+	var interfaces []kubevirtv1.Interface
```

Comment:
Nit: can be also grouped:
```go
var  (
  interfaces []kubevirtv1.Interface
  networks []kubevirtv1.Network
  hasDefault bool
)

Reply:
This code has been refactored, see above.